fleet_operations/__manifest__.py

```diff
@@ -38,16 +38,18 @@
         'views/fleet_service_view.xml',
         'views/fleet_driver_views.xml',
         'views/fleet_extended_view.xml',
-        # 'views/fleet_operation_account.xml',
-        # 'views/fleet_service_view.xml',
         'views/update_pending_history_view.xml',
-        'views/template.xml',
         'views/mail_template.xml',
         'wizard/work_order_reports_view.xml',
         'wizard/xlsx_report_view.xml',
         'wizard/vehicle_change_history_view.xml',
         'wizard/repair_line_summary_view.xml',
```

Comment:
1)version: Instead of 14.0.1.0.0, it should be 15.0.1.0.0
2)website: it should be https not http.
3)remove sequence